### PR TITLE
Core v2 docs update

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description
This PR contains the changes required to the docs for Core v2 (and other things that we forgot to update). Note that this does not document any new APIs, just changing/removing old ones.

I've removed outdated API docs in anticipation of generated API docs (which I will pick back up after this release).

#### Tests
```
improbadoc serve ./docs --experimentz="{\"experiments\":[{\"experiment_name\":\"docs_table_of_contents_v2\",\"conditional\":{\"bool\":{\"enabled\":true}}}]}"
```
For new layout happiness.

#### Documentation
Yes

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.
@ElleEss

### CODE REVIEW

File: docs/content/ecs/authority.md [hunk 1]
```diff
@@ -10,6 +10,16 @@
 
 ## How authority is represented in Unity's ECS
 
+### Authority shared components
+
+For every [SpatialOS component]({{urlRoot}}/content/glossary#spatialos-component) attached to a SpatialOS entity, the SpatialOS GDK for Unity (GDK) attaches a _ECS shared component_ (an ECS `ISharedComponentData`) to the ECS entity. This component is called `ComponentName.ComponentAuthority`, where `ComponentName` is the name of the SpatialOS component.
```
Comment (ElleEss) [inline docs/content/ecs/authority.md]:
```suggestion
For every [SpatialOS component]({{urlRoot}}/content/glossary#spatialos-component) attached to a SpatialOS entity, the GDK attaches an _ECS shared component_ (an ECS `ISharedComponentData`) to the ECS entity. This component is called `ComponentName.ComponentAuthority`, where `ComponentName` is the name of the SpatialOS component.
```

File: docs/content/ecs/authority.md [hunk 2]
```diff
@@ -10,6 +10,16 @@
...
+
+This component contains a single field, `HasAuthority`, a `bool` that indicates whether the worker instance has authority over the SpatialOS component.
+
+> Note that this component does not contain information about `AuthorityLossImminent` notifications. To listen to these, see [Authority reactive components](#authority-reactive-components) below.
```
Comment (ElleEss) [inline docs/content/ecs/authority.md]: Do we say "listen"? Maybe "receive updates on" ? And is it the worker instance which is getting updates?

And do we mean:
```suggestion
> Note that this component does not contain information about `AuthorityLossImminent` notifications. For information on how a worker instance gets updates on these, see [Authority reactive components](#authority-reactive-components) below.
```

File: docs/content/ecs/component-updates.md [hunk 3]
```diff
@@ -49,6 +51,72 @@ public class SendHealthUpdateSystem : ComponentSystem
 }
 ```
 
+#### Component update edge case
+
+There is one edge case that you need to be careful about which is a result of how component changes are tracked and [C# reference type](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/reference-types) semantics.
```
Comment (ElleEss) [inline docs/content/ecs/component-updates.md]: Breaking up a long sentence with a crafty colon.
```suggestion
There is one edge case that you need to be careful about: this is a result of how component changes are tracked and [C# reference type](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/reference-types) semantics.
```

File: docs/content/ecs/component-updates.md [hunk 4]
```diff
@@ -49,6 +51,72 @@ public class SendHealthUpdateSystem : ComponentSystem
...
+
+Simply adding or removing elements to a `List<T>` or a `Dictionary<K, V>` field in a component is not sufficient to trigger a component update. Instead, you need to set the field back to itself to trigger the update. This applies to `List<T>` and `Dictionary<K, V>` fields within types as well.
```
Comment (ElleEss) [inline docs/content/ecs/component-updates.md]: I commented suggesting for an example....before I spotted the excellent example - maybe call out the example?

```suggestion
Simply adding or removing elements to a `List<T>` or a `Dictionary<K, V>` field in a component is not sufficient to trigger a component update. Instead, you need to set the field back to itself to trigger the update, as shown in the example below. This applies to `List<T>` and `Dictionary<K, V>` fields within types as well.
```

File: docs/content/ecs/entity-contracts.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ A reactive component is implemented as a temporary component, and so has the sam
 
 ## Components on ECS Entities that represent SpatialOS entities
```
Comment (ElleEss) [inline docs/content/ecs/entity-contracts.md]: I am not going to comment on the changes not having "worker instance" as it's in all the docs without "instance" but, going forward, if we can say "worker instance" everywhere it's relevant that helps understanding.

And "has authority over" rather than "is authoritative over" also helps understanding, so we always use the same phrase.

File: docs/content/gameobject/interact-spatialos-monobehaviours.md [hunk 6]
```diff
@@ -34,20 +34,19 @@ The requirements depend on which types are marked as required in the Monobehavio
...
+
 The GDK manages the lifecycle of all MonoBehaviours that contain at least
 one field with a `[Require]` attribute. It ensures that these MonoBehaviours are disabled for
 
   * all Prefabs stored inside the `Resources` folder of your Unity project
   * all MonoBehaviours on instantiated GameObjects that have at least one requirement not fulfilled during runtime
-Whenever a MonoBehaviour is disabled, all fields decorated with the `[Require]` attribute are `null` and you can not use them anymore as soon as `OnDisable` is called on that MonoBehaviour.
 
->  `OnTriggerEnter()` and `OnCollisionEnter()` might be called even if the MonoBehaviour is disabled. You need to check whether your required types are available, if you want to use them in these methods. You can verify a field’s availability by checking if that field is equal to null or not. If it is not null, it is safe to use.
+Whenever a MonoBehaviour is disabled, all fields decorated with the `[Require]` attribute are `null` and you can not use them as soon as `OnDisable` is called on that MonoBehaviour.
```
Comment (ElleEss) [inline docs/content/gameobject/interact-spatialos-monobehaviours.md]:
```suggestion
Whenever a MonoBehaviour is disabled, all fields decorated with the `[Require]` attribute are `null` and you cannot use them as soon as `OnDisable` is called on that MonoBehaviour.
```

File: docs/content/gameobject/linking-spatialos-entities.md [hunk 7]
```diff
@@ -15,60 +16,61 @@ This document is a guide on how to link a SpatialOS entity with a GameObject usi
...
- In your project, create a prefab for any SpatialOS entity you want to represent as a GameObject. Where you store the prefab depends on which worker is going to create the GameObject.
-  * For any worker, use the `Resources/Prefabs/Common/` directory.  
+
+The GameObject Creation Feature Module contains an implementation of how to link GameObjects to SpatialOS entities along with a default implementation of how to create GameObjects.
+To enable the GameObject Creation Feature Module in your project:
+
+1. Set up your project to use the GameObject Creation Feature Module.
+  * In your Unity project’s `Packages` directory, locate the Unity Packages manifest `manifest.json`. Add the GameObject Creation Feature Module to the manifest by adding: `"com.improbable.gdk.gameobjectcreation": "file:<path-to-gdk>/workers/unity/Packages/com.improbable.gdk.gameobjectcreation"`
+1. Set up your workers to use the GameObject Creation Feature Module when the [WorkerConnector]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector) initializes systems.
```
Comment (ElleEss) [inline docs/content/gameobject/linking-spatialos-entities.md]:
```suggestion
1. Set up your worker types to use the GameObject Creation Feature Module when the [WorkerConnector]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector) initializes systems.
```
I think this matters here.

File: docs/content/gameobject/linking-spatialos-entities.md [hunk 8]
```diff
@@ -15,60 +16,61 @@ This document is a guide on how to link a SpatialOS entity with a GameObject usi
...
+  * To use the default implementation of GameObject creation, in your implementation of `WorkerConnector.HandleWorkerConnectionEstablished`, call `GameObjectCreationHelper.EnableStandardGameObjectCreation(Worker.World)`.
+1. In your project, create a prefab for any SpatialOS entity you want to represent as a GameObject. Where you store the prefab depends on which worker is going to create the GameObject.
```
Comment (ElleEss) [inline docs/content/gameobject/linking-spatialos-entities.md]:
```suggestion
1. In your project, create a prefab for any SpatialOS entity you want to represent as a GameObject. Where you store the prefab depends on which worker type is going to create the GameObject.
```
Is it worker type or worker instance?? It's not clear here.

File: docs/content/gameobject/world-commands.md [hunk 9]
```diff
@@ -32,145 +32,78 @@ The commands documentation is:
...
 To avoid this, the worker can reserve an entity ID before sending the `CreateEntity` command request. Depending on the command response, you should do the following:
-The command succeeded: The entity got successfully created
-The command failed with the status code `ApplicationError` and the following message: “'Entity reservation failed. The entity with Id <{id}> could not be found. The reservation might have expired or never existed”. If you have used the entity ID that you received from the `ReserveEntityIds` command response for sending the `CreateEntity` commands, then your entity has already been created and you don’t need to retry it anymore.
-The command timed out or another error appeared: retry the command.
-[//]: # (TODO - link to status codes for error messages.)
 
+  * The command succeeded: The entity got successfully created
```
Comment (ElleEss) [inline docs/content/gameobject/world-commands.md]:
```suggestion
  * The command succeeded: The entity got successfully created.
```

File: docs/content/gameobject/world-commands.md [hunk 10]
```diff
@@ -32,145 +32,78 @@ The commands documentation is:
...
+  * The command failed with the status code `ApplicationError` and the following message: “'Entity reservation failed. The entity with Id <{id}> could not be found. The reservation might have expired or never existed”. If you have used the entity ID that you received from the `ReserveEntityIds` command response for sending the `CreateEntity` commands, then your entity has already been created and you don’t need to retry it anymore.
```
Comment (ElleEss) [inline docs/content/gameobject/world-commands.md]:
```suggestion
  * The command failed with the status code `ApplicationError` and the following message: `Entity reservation failed. The entity with Id <{id}> could not be found. The reservation might have expired or never existed`. 
  <br/>If you have used the entity ID that you received from the `ReserveEntityIds` command response for sending the `CreateEntity` commands, then your entity has already been created and you don’t need to retry it anymore.
```
I know you hate BRs but it helps break up the message and the action.

File: docs/content/ecs/component-updates.md [hunk 11]
```diff
@@ -22,10 +22,12 @@ component Health {
 }
 ```
 
-When a SpatialOS entity is [checked out]({{urlRoot}}/content/glossary#checking-out), its components are automatically added to the corresponding ECS entity as part of the entity's check out process.
+When a SpatialOS entity is [checked out]({{urlRoot}}/content/glossary#checking-out), its components are added to the corresponding ECS entity as part of the entity's check out process.
 
 ### How to send a component update
-To send a component update, set the component to the value to be sent. A component update will be constructed and sent at the end of the current update loop.
+
+To send a component update, set the component to the value to be sent. A component update is constructed and sent at the end of the current update loop.
```
Comment (jessicafalk) [inline docs/content/ecs/component-updates.md]:
```suggestion
To send a component update, change the value of at least one field of the component . A component update is constructed and sent at the end of the current update loop.
```

still a bit weird, but maybe a bit clearer?

File: docs/content/ecs/entity-contracts.md [hunk 12]
```diff
@@ -44,19 +43,19 @@ The following guarantees hold for any ECS Entity representing a SpatialOS entity
 ### Guarantees when receiving updates or messages
 Whenever a component update or message is received by the worker, the following holds for the entity that the worker received it for:
 
-  * All updates received are stored as a list in a reactive component called `ReceivedUpdates<T>`, where `T` is the type of the component.
+  * All updates received are stored as a list in a reactive component called `{name of component}.ReceivedUpdates`.
```
Comment (jessicafalk) [inline docs/content/ecs/entity-contracts.md]: do we only have the reactive version of that for ecs?

File: docs/content/gameobject/create-delete-spatialos-entities.md [hunk 13]
```diff
@@ -220,7 +190,5 @@ public class EntityDeletionBehaviour : MonoBehaviour
 #### Getting SpatialOS entity IDs to use for deletion
 You can get the `EntityId` for an entity using these methods:
 
-* To get the `EntityId` for the current GameObject, use `GetComponent<[SpatialOSComponent](link to spatialoscomp docs)>().SpatialEntityId`
-* To get the `EntityId` for a different GameObject, use `SpatialOSComponent.TryGetSpatialOSEntityIdForGameObject(GameObject linkedGameObject, out EntityId entityId)`
-
-> Using `SpatialOSComponent.TryGetSpatialOSEntityIdForGameObject` to access the SpatialOS entity id of another linked GameObject ensures that you receive the correct SpatialOS entity ID.
+* To get the `EntityId` for the current GameObject, use `[Require] private EntityId entityId`.
+* To get the `EntityId` for a different GameObject, use `linkGameObject.GetComponent<LinkedEntityComponent>().EntityId`.
```
Comment (jessicafalk) [inline docs/content/gameobject/create-delete-spatialos-entities.md]: major nit: just to make it absolutely clear 
```suggestion
* To get the `EntityId` for a different GameObject, use `otherGameObject.GetComponent<LinkedEntityComponent>().EntityId`.
```

File: docs/content/gameobject/linking-spatialos-entities.md [hunk 14]
```diff
@@ -15,60 +16,61 @@ This document is a guide on how to link a SpatialOS entity with a GameObject usi
 
 You create SpatialOS entities by creating setting up entity templates. Find out how to do this in the [Entity templates]({{urlRoot}}/content/entity-templates) documentation.
 
-When you are using the MonoBehaviour workflow, you interact with SpatialOS entities in the [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime) by using Readers and Writers for every SpatialOS entity’s component in [schema]({{urlRoot}}/content/glossary#schema). Find out how to use interact with SpatialOS entities in the [Reader and Writers]({{urlRoot}}/content/gameobject/readers-writers).
+When you are using the MonoBehaviour workflow, you interact with SpatialOS entities in the [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime) by using Readers and Writers. Find out how to use interact with SpatialOS entities in the [Reader and Writers]({{urlRoot}}/content/gameobject/readers-writers).
```
Comment (jessicafalk) [inline docs/content/gameobject/linking-spatialos-entities.md]:
```suggestion
When you are using the MonoBehaviour workflow, you interact with SpatialOS entities in the [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime) by using Readers and Writers. Find out how to interact with SpatialOS entities in the [Reader and Writers documentation]({{urlRoot}}/content/gameobject/readers-writers).
```

File: docs/content/gameobject/linking-workers-gameobjects.md [hunk 15]
```diff
@@ -10,24 +10,19 @@ Before reading this document, make sure you are familiar with:
...
 1. Pass the GameObject into the Creation Feature Module using the following methods:
   *  If you are using the default implementation of `GameObjectCreationHelper.EnableStandardGameObjectCreation(Worker.World, YourWorkerGameObject)` if you use the default implementation.
   * GameObjectCreationHelper.EnableStandardGameObjectCreation(Worker.World, new YourEntityGameObjectCreator(), YourWorkerGameObject) if you customized the `IEntityGameObjectCreator` where `YourWorkerGameObject` is the GameObject that should represent your worker.
-
-
-3.  When your game runs,  the Creation Feature Module attaches a GameObject component called `SpatialOSComponent` to the GameObject representing the SpatialOS entity.</br>
-* Note that the `SpatialEntityId` field of the `SpatialOSComponent` is set to 0.</br>
-* See the documentation to find out more about the [`SpatialOSComponent`]({{urlRoot}}/content/gameobject/spatialoscomponent).
-
+1.  When your game runs,  the Creation Feature Module attaches a GameObject component called `LinkedEntityComponent` to the GameObject representing the SpatialOS entity.</br>
```
Comment (jessicafalk) [inline docs/content/gameobject/linking-workers-gameobjects.md]:
```suggestion
1.  When your game runs, the GameObject Creation Feature Module attaches a GameObject component called `LinkedEntityComponent` to the GameObject representing the SpatialOS entity.</br>
```

should we say GameObject Component or MonoBehaviour or script?

File: docs/content/gameobject/linking-workers-gameobjects.md [hunk 16]
```diff
@@ -10,24 +10,19 @@ Before reading this document, make sure you are familiar with:
...
+  * Note that the `SpatialEntityId` field of the `LinkedEntityComponent` is set to 0.</br>
```
Comment (jessicafalk) [inline docs/content/gameobject/linking-workers-gameobjects.md]: what does that mean for users?

File: docs/content/gameobject/sending-receiving-commands.md [hunk 17]
```diff
@@ -60,49 +62,55 @@ The GDK generates the following classes in the `Playground` namespace:
...
-  * `CubeSpawner.Requirable.CommandResponseHandler`
+  * `CubeSpawnerReader`
+  * `CubeSpawnerWriter`
+  * `CubeSpawnerCommandSender`
+  * `CubeSpawnerCommandReceiver`
 
 #### How to send command requests
-The following code snippet provides an examples on how to send a command request.
+
+The following code snippet provides an examples on how to send a command request and register a callback for the response.
```
Comment (jessicafalk) [inline docs/content/gameobject/sending-receiving-commands.md]:
```suggestion
The following code snippet provides an example on how to send a command request and register a callback for the response.
```

File: docs/content/gameobject/sending-receiving-commands.md [hunk 18]
```diff
@@ -136,104 +141,65 @@ public class SpawnCubeCommandRequestHandlerBehaviour : MonoBehaviour
...
-            // Command failed. Handle the failure
-            // The payload will be null in this case.
-            Debug.Log(response.Message);
-        }
-    }
-}
-```
-[//]: # (TODO - split this section into a different doc)
-### API
+The `CommandSender` is used to send command requests, it provides two method per command it is responsible for:
```
Comment (jessicafalk) [inline docs/content/gameobject/sending-receiving-commands.md]:
```suggestion
The `CommandSender` is used to send command requests, it provides two methods per command. They are responsible for:
```

File: docs/content/gameobject/world-commands.md [hunk 19]
```diff
@@ -32,145 +32,78 @@ The commands documentation is:
...
 ## How to send and receive world commands
-We provide the following types for sending and receiving world commands:
+We provide the following type for sending and receiving world commands:
 
-  * `WorldCommands.Requirable.WorldCommandRequestSender`
-  * `WorldCommands.Requirable.WorldCommandResponseHandler`
+  * `WorldCommandSender`
 
-> We do not provide a `WorldCommands.Requirable.WorldCommandRequestHandler` as the command will be directly handled by the SpatialOS Runtime.
+> We do not provide a `WorldCommandReceiver` as the command will be directly handled by the SpatialOS Runtime.
```
Comment (jessicafalk) [inline docs/content/gameobject/world-commands.md]:
```suggestion
> We do not provide a `WorldCommandReceiver` as the SpatialOS Runtime directly handles the command.
```

:D

File: docs/content/gameobject/world-commands.md [hunk 20]
```diff
@@ -32,145 +32,78 @@ The commands documentation is:
...
 
-Both of these objects can be injected without any condition. A MonoBehaviour that requires only these will be enabled as soon as the associated GameObject is created.
+The `WorldCommandSender` can be injected without any condition. A MonoBehaviour that requires only these will be enabled as soon as the associated GameObject is created.
```
Comment (jessicafalk) [inline docs/content/gameobject/world-commands.md]:
```suggestion
The `WorldCommandSender` can be injected without any condition. A MonoBehaviour that requires only this will be enabled as soon as the associated GameObject is created.
```

File: docs/content/gameobject/world-commands.md (hunk 10 above)
Comment (jessicafalk) [reply docs/content/gameobject/world-commands.md]: entity Id instead of entity ID?

File: docs/content/gameobject/world-commands.md [hunk 21]
```diff
@@ -32,145 +32,78 @@ The commands documentation is:
...
-Parameters:
-
-  * `EntityId entityId`: The ID of the SpatialOS entity that you want to delete.
-  * `uint timeoutMillis`: Optional. Specifies the amount of time in milliseconds to wait before the command fails with a timeout status. If not specified, the default of 5 seconds of the underlying Worker SDK is used.
-  * `object context`: Optional. An arbitrary object you can associate with the command. You get this object back along with the response. This is useful when handling the response based on the information contained in this object.
-
-Returns: the request Id for this command as a `long`.
-
-The corresponding response callback is `WorldCommandResponseHandler.OnDeleteEntityResponse`:
+You can use the `SendDeleteEntityCommand` method to request the deletion of a SpatialOS entity given its SpatialOS entity ID. It has the following signature:
```
Comment (jessicafalk) [inline docs/content/gameobject/world-commands.md]: not completely sure, but I think we use ID and Id interchangeably. probably should do a pass to use one or the other

File: docs/content/glossary.md [hunk 22]
```diff
@@ -90,7 +90,7 @@ Code generation automatically occurs when you open the [Unity Project](#unity-pr
 
 ### Connection
 
-Before the [worker](#worker) can interact with the [SpatialOS world](#spatialos-world), the worker must connect to SpatialOS. The connection is established by an instance of the [`Improbable.Gdk.Core.Worker` class]({{urlRoot}}/content/workers/api-worker). See [Connecting to SpatialOS]({{urlRoot}}/content/connecting-to-spatialos) for more information.
+Before the [worker](#worker) can interact with the [SpatialOS world](#spatialos-world), the worker must connect to SpatialOS. The connection is established by an instance of the `Improbable.Gdk.Core.Worker` class. See [Connecting to SpatialOS]({{urlRoot}}/content/connecting-to-spatialos) for more information.
```
Comment (jessicafalk) [inline docs/content/glossary.md]:
```suggestion
Before the [worker](#worker) can interact with the [SpatialOS world](#spatialos-world), the worker must connect to SpatialOS. The connection is established using the `Improbable.Gdk.Core.Worker` class. See [Connecting to SpatialOS]({{urlRoot}}/content/connecting-to-spatialos) for more information.
```

File: docs/content/glossary.md [hunk 23]
```diff
@@ -323,7 +311,7 @@ The Receptionist service allows for a direct connection to the SpatialOS runtime
 
 > Also referred to as "synchronization".
 
-Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by either [standard](#standard-replication) or [custom](#custom-replication) replication.
+Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by either [standard](#standard-replication).
```
Comment (jessicafalk) [inline docs/content/glossary.md]:
```suggestion
Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by using the [standard replication](#standard-replication).
```

File: docs/content/set-up-new-project.md [hunk 24]
```diff
@@ -42,7 +42,7 @@ This triggers the following actions:
   
   > This only happens the first time you open the project or if the required libraries change.
 
-  * Unity [generates code]({{urlRoot}}/content/code-generator) from the [schema]({{urlRoot}}/content/glossary#schema) files defines in your SpatialOS project.
+  * Unity generates code from the [schema]({{urlRoot}}/content/glossary#schema) files defines in your SpatialOS project.
```
Comment (jessicafalk) [inline docs/content/set-up-new-project.md]:
```suggestion
  * Unity generates code from the [schema]({{urlRoot}}/content/glossary#schema) files defined in your SpatialOS project.
```

File: docs/content/workers/workers-in-the-gdk.md [hunk 25]
```diff
@@ -25,10 +25,11 @@ When it connects, the worker creates an ECS world to keep track of all SpatialOS
 
 This means that, along with a SpatialOS game world and SpatialOS entities, there is an ECS world with ECS entities. However, while the SpatialOS world is game-wide and represents all SpatialOS entities, the ECS world is much narrower; it’s worker-specific and represents only the ECS entities which are currently in a [worker’s view]({{urlRoot}}/content/glossary#worker-s-view).   Of course, this means that you have as many ECS worlds in your game as you have workers.
 
-You add definitions of the systems inside your worker using the worker API `worker.World.GetOrCreateManager<YourSystem>()`. (See the [worker API]({{urlRoot}}/content/workers/api-worker) documentation for more information.)
+You add definitions of the systems inside your worker using `worker.World.GetOrCreateManager<YourSystem>()`.
```
Comment (jessicafalk) [inline docs/content/workers/workers-in-the-gdk.md]:
```suggestion
You add the systems to the ECS world using `worker.World.GetOrCreateManager<YourSystem>()`.
```

File: docs/content/ecs/authority.md (hunk 2 above)
Comment (jamiebrynes7) [reply docs/content/ecs/authority.md]: I'd hesitate to use `updates` since its highly associated with component updates

File: docs/content/ecs/component-updates.md (hunk 11 above)
Comment (jamiebrynes7) [reply docs/content/ecs/component-updates.md]: Agreed

File: docs/content/ecs/entity-contracts.md (hunk 12 above)
Comment (jamiebrynes7) [reply docs/content/ecs/entity-contracts.md]: There's an API on the `ComponentUpdateSystem` to get all updates of a type `T`. 

Not documenting these APIs just yet

File: docs/content/gameobject/linking-workers-gameobjects.md (hunk 15 above)
Comment (jamiebrynes7) [reply docs/content/gameobject/linking-workers-gameobjects.md]: Hmmm not sure - I'll see what ours (and Unity's) convention is

File: docs/content/gameobject/linking-workers-gameobjects.md (hunk 16 above)
Comment (jamiebrynes7) [reply docs/content/gameobject/linking-workers-gameobjects.md]: Not a whole lot - just a callout since having a `SpatialEntityId` for something that isn't a SpatialOS entity. 

We could omit it entirely if you prefer

File: docs/content/gameobject/linking-spatialos-entities.md (hunk 8 above)
Comment (jamiebrynes7) [reply docs/content/gameobject/linking-spatialos-entities.md]: Type is correct

File: docs/content/glossary.md (hunk 23 above)
Comment (jamiebrynes7) [reply docs/content/glossary.md]: Not sure if `the` is correct here

Comment (jamiebrynes7) [reply docs/content/glossary.md]: Honestly I'd like to remove this entirely given that there is no standard replication or custom replication - just replication

File: docs/content/ecs/entity-contracts.md (hunk 12 above)
Comment (jessicafalk) [reply docs/content/ecs/entity-contracts.md]: fair enough, do we have a ticket or todo tracking that?

File: docs/content/ecs/authority.md (hunk 2 above)
Comment (jessicafalk) [reply docs/content/ecs/authority.md]: ```> Note that this component does not contain information about `AuthorityLossImminent` notifications. For information on how a worker instance gets notified about these, see [Authority reactive components](#authority-reactive-components) below.```

?

File: docs/content/gameobject/linking-workers-gameobjects.md (hunk 16 above)
Comment (jessicafalk) [reply docs/content/gameobject/linking-workers-gameobjects.md]: no, it's better to mention it. maybe we can add a note that this entity spatial entity is not valid? as it right now sounds like it could be a valid spatial entity

File: docs/content/ecs/entity-contracts.md (hunk 12 above)
Comment (jamiebrynes7) [reply docs/content/ecs/entity-contracts.md]: Yes - https://improbableio.atlassian.net/browse/UTY-1683

File: docs/content/build.md [hunk 26]
```diff
@@ -61,6 +61,7 @@ This starts the build process for the worker and deployment type (cloud or local
 You can now find the built-out workers in `build/assembly/worker` in the root of the SpatialOS GDK for Unity project.
 
 #### TIP: Speed up development iteration with worker quick-run
+
  When you are developing with the GDK, you don't need to build out workers all the time, you can use quick-run to run multiple workers in your Unity Editor using Ctrl+L (Windows) or Cmd+L (Mac).
```
Comment (samiwh) [inline docs/content/build.md]: Not part of this PR. But what is this seems like nonsense.

File: docs/content/connecting-to-spatialos.md [hunk 27]
```diff
@@ -5,14 +5,14 @@
...
 
-If you are using the [MonoBehaviour workflow or the ECS workflow]{{urlRoot}}/content/intro-workflows-spatialos-entities), you can use the [Worker Connector]({{urlRoot}}/content/gameobject/linking-spatialos-entities) or the [Worker API]({{urlRoot}}/content/workers/api-worker) to set up your workers (server-workers and client-workers) to connect to SpatialOS. Upon successfully connecting to the SpatialOS Runtime, your worker stores a `Connection` object.
+If you are using the [MonoBehaviour workflow or the ECS workflow]{{urlRoot}}/content/intro-workflows-spatialos-entities), you can use the [Worker Connector]({{urlRoot}}/content/gameobject/linking-spatialos-entities) or the `Worker` to set up your workers (server-workers and client-workers) to connect to SpatialOS. Upon successfully connecting to the SpatialOS Runtime, your worker stores a `Connection` object.
```
Comment (samiwh) [inline docs/content/connecting-to-spatialos.md]: I think this might be fine for now, but ideally we don't want users to have to know there is a `Connection` object as it should be completely hidden.

File: docs/content/ecs/entity-contracts.md [hunk 28]
```diff
@@ -23,7 +23,7 @@ A reactive component is implemented as a temporary component, and so has the sam
 
 ## Components on ECS Entities that represent SpatialOS entities
 
-For each SpatialOS entity that a [worker]({{urlRoot}}/content/workers/workers-in-the-gdk) checks out, the `SpatialOSReceiveSystem` creates an ECS Entity for that worker’s ECS world.
+For each SpatialOS entity that a [worker]({{urlRoot}}/content/workers/workers-in-the-gdk) checks out, the `EcsViewSystem` creates an ECS Entity for that worker’s ECS world.
```
Comment (samiwh) [inline docs/content/ecs/entity-contracts.md]: Do we need to say which system? That seems like an implementation detail until we offer users the ability to do it with arbitrary ones.

File: docs/content/ecs/logging.md [hunk 29]
```diff
@@ -13,7 +13,7 @@ All workers use the `ForwardingDispatcher` by default in the Playground. If you
 
 ## Using the ILogDispatcher
 
-You can access the dispatcher through the [WorkerSystem]({{urlRoot}}/content/workers/api-worker-system). The dispatcher provides a single `HandleLog` function, which takes two arguments:
+You can access the dispatcher through the `WorkerSystem`. The dispatcher provides a single `HandleLog` function, which takes two arguments:
```
Comment (samiwh) [inline docs/content/ecs/logging.md]: No you can't :P

File: docs/content/gameobject/world-commands.md (hunk 21 above)
Comment (samiwh) [reply docs/content/gameobject/world-commands.md]: I always use ID. Id seems wrong unless referring to the Freudian concept of the part of you that contains instinctive drives.

File: docs/content/gameobject/world-commands.md [hunk 30]
```diff
@@ -32,145 +32,79 @@ The commands documentation is:
...
 
-  * `ReceivedResponse`: The data associated with the incoming response stored inside a [`ReceivedResponse`]({{urlRoot}}/content/world-component-commands-requests-responses#receivedresponse) struct.
+Parameters:
 
-The `DeleteEntity.ReceivedResponse` struct contains the same fields as any `ReceivedResponse` struct. However, in this case, the `EntityId` field represents the entity ID of the deleted entity. If the command failed, this field is null.
+  * `WorldCommands.DeleteEntity.Request request`: The command request payload.
+  * `Action<WorldCommands.DeleteEntity.ReceivedResponse> callback`: Optional. A callback that will be called when the command response is received.
 
->  Do not manually delete GameObjects representing entities after sending a `DeleteEntity` command. [The GDK does this for you]({{urlRoot}}/content/gameobject/linking-spatialos-entities#the-creation-feature-module).
+>  Do not manually delete GameObjects representing entities after sending a `DeleteEntity` command. [You should wait until you receive a callback on ]({{urlRoot}}/content/gameobject/linking-spatialos-entities#the-creation-feature-module).
```
Comment (samiwh) [inline docs/content/gameobject/world-commands.md]: Actually you can do this if you want to.

File: docs/content/ecs/authority.md (hunk 2 above)
Comment (ElleEss) [reply docs/content/ecs/authority.md]: LGTM!!! 
Completely understand about updates.

File: docs/content/gameobject/linking-workers-gameobjects.md (hunk 15 above)
Comment (paulbalaji) [reply docs/content/gameobject/linking-workers-gameobjects.md]: how about
```suggestion
1.  When your game runs, the GameObject Creation Feature Module attaches a MonoBehaviour called `LinkedEntityComponent` to the GameObject representing the SpatialOS entity.</br>
```

**Comment by paulbalaji** [comment]
Laura gave a second OK for this, merging 🎉
